src/ui/public/chrome/directives/app_switcher/app_switcher.less

```diff
@@ -107,6 +107,10 @@ body { overflow-x: hidden; }
         margin-top: 8px;
         filter: invert(100%);
       }
+      > svg {
+        height: 18px;
+        margin-top: 10px;
+      }
```

Comment:
The directive should work without needing additional styles applied in an external stylesheet.